test/test-utils.sh

```diff
@@ -315,11 +315,11 @@ function get_permissions() {
 
 function check_content_type() {
     local INFO_STR
-    INFO_STR=$(aws_cli s3api head-object --bucket "${TEST_BUCKET_1}" --key "$1")
-    if [[ "${INFO_STR}" != *"$2"* ]]
+    INFO_STR=$(aws_cli s3api head-object --bucket "${TEST_BUCKET_1}" --key "$1" | jq -r .ContentType)
+    if [[ "${INFO_STR}" != "$2" ]]
```

Comment:
I think the double brackets are no longer needed.

Reply:
Done.

Reply:
Thanks.